helm_deploy/hmpps-workload-history/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: '1.0'
+description: A Helm chart for Kubernetes
+name: hmpps-workload-history
+version: 0.2.0
+dependencies:
+  - name: generic-service
+    version: 1.0.10
+    repository: https://ministryofjustice.github.io/hmpps-helm-charts
+  - name: generic-prometheus-alerts
```

Comment:
The other chart doesn't have these alerts and I don't think we need them until this becomes an API or whatever